lib/Driver/WindowsToolChains.cpp

```diff
@@ -168,5 +168,8 @@ toolchains::Windows::constructInvocation(const LinkJobAction &job,
   Arguments.push_back(
       context.Args.MakeArgString(context.Output.getPrimaryOutputFilename()));
 
-  return {Clang, Arguments};
+  InvocationInfo II{Clang, Arguments};
+  II.allowsResponseFiles = true;
```

Comment:
OK, this is the third time I'm seeing the setting to true. Is there something in common that the code should be reifying?

Reply:
Nope. It's all based on what other tool is getting invoked, which is not necessarily the same across each toolchain and certainly not the same across each job.

Reply:
That is, this *is* the customization point, so that the rest of the code can treat everything the same.

Reply:
The comment would read: `// clang supports response files`. Given the way in which the code is written, such a comment would be redundant.

Reply:
I am still with Austin on this. If the executable specified in the InvocationInfo constructor supports response files, you can put `true` here. I guess the doc comment for `allowsResponseFiles` could say that a little more clearly, but it doesn't belong at the use sites.

Reply:
I updated the comment in `include/swift/Driver/ToolChain.h`.

Reply:
Thank you for updating the comment. In thinking about this item, the comment you added is good and it is sufficient. I'm not asking for the following, but if I were to try to make it even clearer, I would use a one-line intention-revealing function. But, never mind.